safe-global/web-core
# fix: Adds icon to empty address book

## What it solves

Resolves #1192 

## How this PR fixes it

Displays an illustration if the address book is empty

## How to test it

1. Open a safe without any address book entries
2. Observe an icon in the center

## Screenshots

<img width="749" alt="Screenshot 2022-12-22 at 13 30 31" src="https://user-images.githubusercontent.com/5880855/209134898-71148d27-069a-449d-9fe2-7eff0e7936fc.png">

<img width="749" alt="Screenshot 2022-12-22 at 13 30 40" src="https://user-images.githubusercontent.com/5880855/209134928-49a99f4a-1432-4fe9-bbcf-20e79b8303fc.png">
